config/cmd/cmd.go

```diff
@@ -254,6 +254,11 @@ var (
 			EnvVars: []string{"MICRO_AUTH_PRIVATE_KEY"},
 			Usage:   "Private key for JWT auth (base64 encoded PEM)",
 		},
+		&cli.StringSliceFlag{
+			Name:    "auth_excludes",
```

Comment:
auth_exclude

Reply:
Other slice CLI flags used plurals for var names, but I'm happy to amend